tests/yam/agama/auto.pm

```diff
@@ -7,16 +7,25 @@
 use base 'y2_installbase';
 use strict;
 use warnings;
+use serial_terminal;
 
 use testapi 'assert_screen';
 
 
 sub run {
     assert_screen('agama-main-page', 120);
     assert_screen('agama-installing', 60);
+    return 42;
 
     my @tags = ("welcome-to", "login");
     assert_screen \@tags, 960;
 }
 
+sub post_fail_hook {
```

Comment:
post_fail_hook should be a new base class as suggested in the ticket, we can have our own new base class that we can use for agama to retrieve logs. With Agama we don't need to inherit all the mess with retrieving those too many logs that  are not useful, we can add progressively more logs depending on the bugs we will find in this new product.